mlir/test/Target/LLVMIR/Import/intrinsic.ll

```diff
@@ -60,15 +60,6 @@ define void @ldexp_test(float %0, <8 x float> %1, i32 %2) {
   ret void
 }
 
-; CHECK-LABEL:  llvm.func @frexp_test
-define void @frexp_test(float %0, <8 x float> %1) {
-  ; CHECK:  llvm.intr.frexp(%{{.*}}) : (f32) -> !llvm.struct<(f32, i32)>
-  %4 = call { float, i32 } @llvm.frexp.f32.i32(float %0)
-  ; CHECK:  llvm.intr.frexp(%{{.*}}) : (vector<8xf32>) -> !llvm.struct<(vector<8xf32>, i32)>
-  %5 = call { <8 x float>, i32 } @llvm.frexp.v8f32.i32(<8 x float> %1)
-  ret void
-}
-
```

Comment:
Why does this disappear? It makes sense to still test it, just produce a different type.

Reply:
That got lost during rebasing.